### PR TITLE
Reformatting Viralrecon-service documentation

### LINKED ISSUE #45: Complete documentation for viralrecon
- Viralrecon
    - [ ] Documentation (markdown)

### PR DESCRIPTION
**Pull request by jaimeozaez**
Renaming SARS-CoV-2-service onto Viralrecon-service.
Reformatting bioinformatic analisys steps.
closes #45

### CODE REVIEW

File: docs/Service-portfolio.md [hunk 1]
```diff
@@ -44,6 +44,37 @@ Genomic Data Analysis:
...
   - Bacteria: Core genome or whole genome Multi-Locus Sequence Typing analysis (cg/wgMLST) (TODO wgmlst_chewbbaca)
   - Viral: Genomic reconstruction, variant calling and _de novo_ assembly ([viralrecon](https://github.com/BU-ISCIII/buisciii-tools/tree/develop/bu_isciii/templates/viralrecon))
+    - Viralrecon is a bioinformatics analysis pipeline used to perform assembly and intrahost/low-frequency variant calling for viral samples. The pipeline supports both Illumina and Nanopore sequencing data. For Illumina short-reads the pipeline is able to analyse metagenomics data typically obtained from shotgun sequencing (e.g. directly from clinical samples) and enrichment-based library preparation methods (e.g. amplicon-based: ARTIC SARS-CoV-2 enrichment protocol; or probe-capture-based). For Nanopore data the pipeline only supports amplicon-based analysis obtained from primer sets created and maintained by the ARTIC Network. Some examples of viruses analyzed with this pipeline are SARS-CoV-2, mumps virus, monkeypox virus, West Nile virus, etc.
```
Comment (saramonzon) [inline docs/Service-portfolio.md:47]: remove the `-` so the descripcion is not a list

File: docs/Service-portfolio.md [hunk 2]
```diff
@@ -44,6 +44,37 @@ Genomic Data Analysis:
...
+
+    - <details>
```
Comment (saramonzon) [inline docs/Service-portfolio.md:49]: same, check the plasmidID example

File: docs/Service-portfolio.md [hunk 3]
```diff
@@ -44,6 +44,37 @@ Genomic Data Analysis:
...
+
+      * **PRIMERS BED FILE**.
+      In case of amplicon-based method, we need to provide a BED file with primer coordinates for the mapping step.
+
+
+      * **PRIMERS FASTA FILE**.
+      Additionally, a fasta file will be necessary if de novo assembly is requested.
+
+
+      * **VIRALRECON_INPUT.xlsx**
```
Comment (saramonzon) [inline docs/Service-portfolio.md:64]: add this template to an asset folders in BU-ISCIII repo and create a link to the template file.

File: docs/Viralrecon-service.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+> Note: If the resolution ID is not specified, it will be requested via the prompt.
+
+If the service configuration is correct and the sequences are located in `/srv/fastq_repo`, move inside the newly created folder at `/data/bi/services_and_colaborations/CNM/virology/`. Check the `/RAW` folder to verify that symbolic links have been correctly created for all service samples.
+
+Move to `/ANALYSIS`. Configure the `samples_ref.txt` file according to the service requirements (samples, reference genomes, hosts, etc.). Check the lablog and execute it.
+
+    $ bash lablog_viralrecon
+
+Este script consulta al usuario a través del prompt el tipo de análisis que va a llevarse a cabo (AMPLICONS o METAGENOMICS), establece los archivos de configuración de la carpeta `../DOC` conforme a esta decisión y crea una carpeta por cada hospedador que haya sido especificado en `samples_ref.txt` (normalmente solo 1).
```
Comment (saramonzon) [inline docs/Viralrecon-service.md:24]: spanish!

Comment (jaimeozaez) [reply docs/Viralrecon-service.md:24]: Done

File: docs/Service-portfolio.md (hunk 1 above)
Comment (jaimeozaez) [reply docs/Service-portfolio.md:47]: Done

File: docs/Service-portfolio.md (hunk 2 above)
Comment (jaimeozaez) [reply docs/Service-portfolio.md:49]: Done

File: docs/Service-portfolio.md (hunk 3 above)
Comment (jaimeozaez) [reply docs/Service-portfolio.md:64]: Done

File: docs/Service-portfolio.md [hunk 5]
```diff
@@ -58,6 +58,39 @@ Genomic Data Analysis:
...
   - Viral: Genomic reconstruction, variant calling and _de novo_ assembly ([viralrecon](https://github.com/BU-ISCIII/buisciii-tools/tree/develop/bu_isciii/templates/viralrecon))
+    
+    Viralrecon is a bioinformatics analysis pipeline used to perform assembly and intrahost/low-frequency variant calling for viral samples. The pipeline supports both Illumina and Nanopore sequencing data. For Illumina short-reads the pipeline is able to analyse metagenomics data typically obtained from shotgun sequencing (e.g. directly from clinical samples) and enrichment-based library preparation methods (e.g. amplicon-based: ARTIC SARS-CoV-2 enrichment protocol; or probe-capture-based). For Nanopore data the pipeline only supports amplicon-based analysis obtained from primer sets created and maintained by the ARTIC Network. Some examples of viruses analyzed with this pipeline are SARS-CoV-2, mumps virus, monkeypox virus, West Nile virus, etc.
+
+    <details>
+    <summary>Required information for service request</summary>
+    <br>
+    For the correct performance of the pipeline, it is necessary to provide some input documents:
+
+    ---
```
Comment (svarona) [inline docs/Service-portfolio.md:69]: Remove these separating lines

File: docs/Service-portfolio.md [hunk 6]
```diff
@@ -58,6 +58,39 @@ Genomic Data Analysis:
...
+
+    * **PRIMERS BED FILE**.
```
Comment (svarona) [inline docs/Service-portfolio.md:71]: Put the name of the files in CamelCaseWords or normal Capitalization, don't use all upper case for this.

File: docs/Service-portfolio.md [hunk 7]
```diff
@@ -58,6 +58,39 @@ Genomic Data Analysis:
...
+    In case of amplicon-based method, we need to provide a BED file with primer coordinates for the mapping step.
+
+
+    * **PRIMERS FASTA FILE**.
+    Additionally, a fasta file will be necessary if de novo assembly is requested.
+
+
+    * **[VIRALRECON_INPUT.xlsx](https://github.com/BU-ISCIII/BU-ISCIII/blob/main/docs/assets/input_datasets/viralrecon/viralrecon_input.xlsx)**
```
Comment (svarona) [inline docs/Service-portfolio.md:78]: Use the same filename

**Pull request merged**